R/cv_estimate_rt.R

```diff
@@ -102,7 +102,11 @@ cv_estimate_rt <- function(
 
     pred_observed_counts <- interp_rt * wpc[test_idx]
     score <- colMeans(err_fun(observed_counts[test_idx], pred_observed_counts))
-    cvall[f,] <- score
+    tryCatch(cvall[f,] <- score,
```

Comment:
Let's put the result in and return. But also print a warning. Use `cli::cli_warn()`.